week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,42 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
```

Comment:
Nice! how can you make your code shorter though? Remember that just `num < 0` returns a boolean and that's what you want this function to return too!